tests/utils/proxy.go

```diff
@@ -42,8 +41,5 @@ func DeployTransparentUpgradeableProxy[T any](
 	proxyAdmin, err := proxyadmin.NewProxyAdmin(proxyAdminEvent.NewAdmin, subnet.RPCClient)
 	Expect(err).Should(BeNil())
 
-	contract, err := newInstance(proxyAddress, subnet.RPCClient)
-	Expect(err).Should(BeNil())
-
-	return proxyAddress, proxyAdmin, contract
+	return proxyAddress, proxyAdmin
```

Comment:
The target `T` contract instance can be constructed from the `proxyAddress` by the caller.